core/trino-parser/src/main/java/io/trino/sql/tree/Property.java

```diff
@@ -20,25 +20,41 @@
 import java.util.Optional;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
 public class Property
         extends Node
 {
     private final Identifier name;
-    private final Expression value;
+    private final Optional<Expression> value;   // empty iff the value is set to DEFAULT
+
+    /** Constructs an instance representing a property whose value is set to DEFAULT */
```

Comment:
this is not a valid javadoc, it should be:
```
/**
 * xxx
 */
```

Reply:
It is valid javadoc. But anyway, I have formatted it in the way you described.